src/Lens/LensTypes.sol

```diff
@@ -218,8 +218,10 @@ struct OracleDetailedInfo {
 struct EulerRouterInfo {
     address governor;
     address fallbackOracle;
-    address[] resolvedOracles;
```

Comment:
Is this ABI-breaking?

Reply:
Oh I guess *this* particular upgrade will be at an entirely new address anyway, it's *future* updates that we need to worry about!